src/components/select/Select.js

```diff
@@ -61,10 +62,14 @@ const Select = ({
                             <span className="select-overlay" />
                         </span>
                         {infoTooltip && (
-                            <Tooltip position="middle-right" text={infoTooltip}>
-                                <span className="tooltip-icon-container">
+                            <Tooltip isShown={infoTooltipIsOpen} position="middle-right" text={infoTooltip}>
+                                <PlainButton
+                                    type="button"
+                                    className="tooltip-icon-container"
+                                    onClick={() => setInfoTooltipIsOpen(!infoTooltipIsOpen)}
+                                >
                                     <IconInfo height={16} width={16} {...infoIconProps} />
```

Comment:
Can we give a default title or aria-label to the icon so that it defaults to being accessible for screen readers?

Reply:
what about the other tooltip-icons that are not `Select` components
like this one https://git.dev.box.net/Box/EndUserApp/blob/master/src/components/pages/account-sharing-page/components/AppNotificationsSection.js#L27